apps/templates/resource_download.html

```diff
@@ -50,10 +50,10 @@ <h2>{% trans 'Windows Remote application publisher tools' %}</h2>
     </div>
 
     <div class="group">
-        <h2>JumpServer {% trans 'Offline video player' %} v0.1.9</h2>
+        <h2>JumpServer {% trans 'Offline video player' %} v0.2.1</h2>
         <ul>
-            <li><a href="/download/public/JumpServer.Video.Player-{{ VIDEO_PLAYER_VERSION }}.dmg">jumpserver-video-player.dmg</a></li>
-            <li><a href="/download/public/JumpServer.Video.Player.Setup.{{ VIDEO_PLAYER_VERSION }}.exe">jumpserver-video-player.exe</a></li>
+            <li><a href="/download/public/JumpServerVideoPlayer-{{ VIDEO_PLAYER_VERSION }}.dmg">jumpserver-video-player.dmg</a></li>
+            <li><a href="/download/public/JumpServerVideoPlayer-{{ VIDEO_PLAYER_VERSION }}.exe">jumpserver-video-player.exe</a></li>
         </ul>
     </div>
 </div>
```

Comment:
There does not seem to be any significant differences that would warrant further examination. The provided HTML markup doesn't exhibit any errors or potential issues, which could affect the display of content on web platforms like GitBook for example. However, since this is a static file, direct modifications may not yield noticeable improvements in terms of user experience. Nonetheless, if you're working with dynamic files on web servers (e.g., WordPress), there might be better practices involving server-side scripting or API integration tailored towards rendering these assets dynamically based on the client's needs rather than hard-coding into the templates.